javascript/src/seedsource/containers/Map.jsx

```diff
@@ -605,10 +608,41 @@ class Map extends React.Component {
         }
     }
 
+    updateVectorLayers(layers) {
+        let numLayersToAdd = layers.length - this.displayedVectorLayers.length
+        if (this.displayedVectorLayers.map(layer => layer._url).toString() === layers.map(layer => layer.urlTemplate).toString()) {
+            return
+        }
+
+        let rewriteLeafletVectors = () => {
+            if (layers.length) {
+                layers.forEach((layer, index) => {
+                    this.displayedVectorLayers[index].setUrl(layer.urlTemplate)
+                        .setZIndex(layer.zIndex)
+                })
+            }
+        }
+
+        if (numLayersToAdd > 0) {
+            while (numLayersToAdd > 0) {
+                let newVectorLayer = L.vectorGrid.protobuf("http://localhost:3333/services/seedzones/ca_91/tiles/{z}/{x}/{y}.png", {zIndex: 1, opacity: 1}).addTo(this.map)
```

Comment:
Make sure this URL is configurable. We'll need a different URL on production.

Reply:
Also, I wouldn't actually store it in the config file (contrary to my previous statement). Instead, I would store it in `config.json`, put it into a setting in `settings.py` and then inject that into the template in the `SS_CONFIG` object in the template. Just like `title` and `staticRoot` are. That way, it's easy to configure per-environment with `config.json`.

Reply:
You will also need to modify the `ToolView` to pass the setting along to the template.